lua/main_config.lua

```diff
@@ -40,19 +40,24 @@ function gatekeeper_init()
 	local gt_conf
 
 	if gatekeeper_server == true then
-		local n_lcores = 2
-		local gk_lcores_tbl =
+		local n_gk_lcores = 2
+		local n_sol_lcores_per_socket = 1
+		local lcores_tbl =
 			staticlib.alloc_lcores_evenly_from_all_numa_nodes(numa_table,
-				n_lcores, 0)
-		local gk_lcores = staticlib.convert_numa_table_to_array(gk_lcores_tbl)
-		local sol_lcore = staticlib.alloc_an_lcore(aux_numa_table)
+				n_gk_lcores, n_sol_lcores_per_socket)
+		local gk_lcores = staticlib.convert_numa_table_to_array(
+			staticlib.alloc_lcores_evenly_from_all_numa_nodes(lcores_table,
+				n_gk_lcores, 0))
+		local sol_lcores = staticlib.convert_numa_table_to_array(lcores_table)
 		local ggu_lcore = staticlib.alloc_an_lcore(aux_numa_table)
```

Comment:
Style: move the line above between the lines snown below to group all the GGU initialization:
```Lua
local gguf = require("ggu")
local ggu_conf = gguf(net_conf, gk_conf, ggu_lcore)
```